CLAUDE.md

```
@@ -0,0 +1,79 @@
+
+# Lint (requires golangci-lint)
+make lint
+
+# Run locally (HTTP on :8080)
+go run .
+# stdio mode for MCP clients:
+go run . --transport=stdio
+```
+
+Credentials are read from `.env` at startup (via `godotenv.Load()`):
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)